src/material-experimental/mdc-form-field/directives/error.ts

```diff
@@ -22,11 +22,16 @@ export const MAT_ERROR = new InjectionToken<MatError>('MatError');
   selector: 'mat-error',
   host: {
     'class': 'mat-mdc-form-field-error mat-mdc-form-field-bottom-align',
-    'role': 'alert',
+    'aria-atomic': 'true',
+    '[attr.aria-live]': 'ariaLive',
```

Comment:
Could you add a comment here with some of the background on how we got to this approach with  `aria-live`? I want to avoid a situation in the future like we've had with the `role="alert"` now where we don't remember why we landed on that particular approach.

Reply:
added a brief comment about this, PTAL and let me know if there is anything missing / that you'd like me to change.